Kickstarter-iOS/Views/Controllers/DeprecatedCheckoutViewController.swift

```diff
@@ -7,16 +7,16 @@ import SafariServices
 import Stripe
 import UIKit
 
-internal final class CheckoutViewController: DeprecatedWebViewController {
+internal final class DeprecatedCheckoutViewController: DeprecatedWebViewController {
```

Comment:
`DeprecatedWebViewController` has been around forever - wondering when we were meant to remove that 😅